frontend/src/components/SportIcons.tsx

```diff
@@ -0,0 +1,111 @@
+function BaseIcon(props) {
```

Comment:
I think it would be useful to break these icons out into separate files and make use of an `index.ts` file to export them all from a directory.

```
// src/components/icons/BaseIcon.tsx
function BaseIcon(props) {
...
}

export default BaseIcon;
```

```
// src/components/icons/Baseball.tsx
import BaseIcon from "./BaseIcon";

function Baseball(props) {
...
}

export default Baseball;
```

```
//  src/components/icons/index.ts
export { default } from "./Baseball"
...
```

```
// src/pages/Baseball.tsx
import { Baseball } from "../components/icons";

...
```